packages/features/settings/layouts/SettingsLayout.tsx

```diff
@@ -520,23 +520,9 @@ const SettingsSidebarContainer = ({
                                     alt={otherTeam.name || "Team logo"}
                                   />
                                   <p className="w-1/2 truncate">{otherTeam.name}</p>
-                                  {!otherTeam.accepted && otherTeam.userId === session.data?.user.id && (
```

Comment:
Don't we need to show this now?

Reply:
The thing is if you are invited to "other team" it will no longer appear on your Other team list.